### PR TITLE
Feat/int 1184 create order submit button

### PR DESCRIPTION
**Pull request by joyet-simon**

### CODE REVIEW

**Comment by joyet-simon** [comment]
#38

File: cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js [hunk 1]
```diff
@@ -169,13 +171,46 @@ function getUserProfile(email) {
...
+    Transaction.wrap(function () {
+        OrderMgr.failOrder(order, true);
+    });
+    return order;
+}
+
 /**
  * create an order from a Basket
- * @param {string} almaPaymentMethod payment metyhod ID
+ * @throw Error
```
Comment (syjust-alma) [inline cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js:210]:
```suggestion
 * @throws Error
```

File: cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js [hunk 2]
```diff
@@ -169,13 +171,46 @@ function getUserProfile(email) {
...
  * @returns {dw.order.Order} the created order
- * @throws Error
```
Comment (syjust-alma) [inline cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js:176]:
```suggestion
 * @throws Error
```

Comment (syjust-alma) [reply cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js:176]: actually, it throws

File: cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml [hunk 3]
```diff
@@ -193,7 +195,7 @@
             pm.addEventListener("click", (event) => {
                 const isAlmaTab = event.target.getAttribute("href") === '#alma-content';
                 if (isAlmaTab) {
-                    checkoutBtn.setAttribute('disabled', true);
+                    // checkoutBtn.setAttribute('disabled', true);
```
Comment (syjust-alma) [inline cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml:198]: why ?

**Review by syjust-alma — COMMENTED** [review]
little comments

File: cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml [hunk 4]
```diff
@@ -30,62 +29,85 @@
...
+            document
+                .querySelector('.payment-options')
+                .addEventListener(
+                    'click',
+                    () => {
+                        document
+                            .querySelector('.submit-payment')
+                            .removeEventListener('click', checkoutFragmentCall)
+                    }
+                )
```
Comment (syjust-alma) [inline cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml:109]: to refactor / enhance

File: cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml (hunk 3 above)
Comment (syjust-alma) [reply cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml:198]: so remove this part of code (to check together)

**Pull request merged**